pkgs/misc/tmux-plugins/default.nix

```diff
@@ -216,22 +216,23 @@ in rec {
 
   extrakto = mkTmuxPlugin {
     pluginName = "extrakto";
-    version = "unstable-2021-04-04";
+    version = "0-unstable-2024-08-26";
```

Comment:
Nit: the commit is from Aug 25, so the version should be `unstable-2024-08-25` https://github.com/laktak/extrakto/commit/bf9e666f2a6a8172ebe99fff61b574ba740cffc2

Just curious: why `0-`?

Reply:
I read [this doc](https://github.com/NixOS/nixpkgs/tree/master/pkgs#versioning), and it say versioning must start with digit. I just follows the example under the third point.

On my page, GitHub say `Commits on Aug 26, 2024`, so maybe it is timezone  issue?

Reply:
Idk how to read the right time tho